our-app/src/components/unauthenticated/SignupForm.js

```diff
@@ -19,7 +19,8 @@ const SignupForm = (props) => {
         }
 
         post('/user', user)
-            .then(_ => {
+            .then(id => {
+                console.log(id);
```

Comment:
Agreed, something that came up with me and Justin was returning the complete user object (user information and a generated user _id from mongo) versus just returning the generated user _id because the client has everything already when it goes to submit the new user. Do we want to do that for efficiency or just return the whole user object so we dont have a situation where a user object contained in the client state is missing it's corresponding id? Or is that something that is just so niche we dont have to worry about it?

Reply:
Whatever is quickest and works between the both of you.